leobot_description/robot/robot.urdf.xacro

```diff
@@ -15,6 +15,26 @@
     <xacro:cylinder_inertia mass="10" radius="0.25" height="1.0"/>
   </link>
 
+  <!-- Head link -->
+  <link name="head_link">
+    <visual>
+      <xacro:cylinder_geometry radius="0.15" height="0.1"/>
+      <xacro:navajo_white_material/>
+    </visual>
+    <xacro:cylinder_collision radius="0.15" height="0.1"/>
+    <xacro:cylinder_inertia mass="1" radius="0.15" height="0.1"/>
+  </link>
+
+  <joint name="head_link_joint" type="revolute">
+    <origin rpy="0 0 0" xyz="0 0 0.525"/>
+    <axis xyz="0 0 1" />
+    <parent link="base_link"/>
+    <child link="head_link"/>
+    <limit effort="10" velocity="10" lower="-1.57" upper="1.57" />
```

Comment:
Please use XACRO expression for PI related constants like here http://wiki.ros.org/xacro#Math_expressions